core/eth/reader.go

```diff
@@ -210,6 +210,18 @@ func (t *Reader) updateContractBindings(blsOperatorStateRetrieverAddr, eigenDASe
 		}
 	}
 
+	paymentVaultAddr, err := contractEigenDAServiceManager.PaymentVault(&bind.CallOpts{})
+	if err != nil {
```

Comment:
let's not fail and account for the case contract isn't deployed like threshold registry above

Reply:
okay, edited to just log